cmd/pg/wal_prefetch.go

```diff
@@ -20,7 +20,10 @@ var walPrefetchCmd = &cobra.Command{
 		if err != nil {
 			tracelog.ErrorLogger.FatalError(err)
 		}
-		internal.HandleWALPrefetch(uploader, args[0], args[1])
+		err = internal.HandleWALPrefetch(uploader, args[0], args[1])
```

Comment:
You should handle error in handler not in cobra command.

Reply:
Fine remark, fixed it